web/admin/src/Admin.js

```diff
@@ -116,10 +116,18 @@ export default class Admin extends PureComponent {
   }
 
   isDisplayable = (sessionId) => {
-    const currentSession = this.state.sessions[sessionId]
-    const dup = Object.values(this.state.sessions).find(i => i.name.toLowerCase() === currentSession.name.toLowerCase() && currentSession.id !== i.id)
-    const isDisplayable = currentSession.name.length > 0 && !dup
-    return isDisplayable
+    if (this.state.isFocus) return true
```

Comment:
Style: don't combine braced/unbraced clauses in the same `if`/`else`.  However, the `else` is not required, since everything inside it will execute only if we don't return on line 119.